subgraph.yaml

```diff
@@ -9,11 +9,11 @@ features:
 dataSources:
   - kind: ethereum/contract
     name: Factory
-    network: mainnet
```

Comment:
how are we handling the config for the yaml file here? and do we want to check in this base change?